src/main/java/zean/Storage.java

```diff
@@ -0,0 +1,112 @@
+package zean;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import zean.exception.DukeException;
+import zean.task.Task;
+
+/**
+ * The class that deals with loading tasks from the file and saving tasks in the file.
+ *
+ * @author Zhong Han
+ */
+public class Storage {
+
+    private File f;
+
+    private String filePath;
+
+    /**
+     * Empty constructor for storage.
+     * For the purpose of testing.
+     */
+    public Storage() {
```

Comment:
if I'm not wrong if the constructor is empty you can choose to omit it, do double check on this thanks!